Gopkg.toml

```diff
@@ -1,30 +0,0 @@
-# Gopkg.toml:
-# this `dep` file is used only to lock Tracer dependencies. It's not meant to be 
-# used by end users so no integrations dependencies must be added here. If you update 
-# or add a new dependency, remember to commit the `vendor` folder. To prepare
-# your development environment, remember to use `rake init` instead.
-
-# ignore integrations dependencies
-ignored = [
-  "github.com/opentracing/*",
-  "github.com/cihub/seelog",
-  "github.com/gin-gonic/gin",
-  "github.com/go-redis/redis",
-  "github.com/go-sql-driver/mysql",
-  "github.com/gocql/gocql",
-  "github.com/gorilla/mux",
-  "github.com/jmoiron/sqlx",
-  "github.com/lib/pq",
-  "google.golang.org/grpc",
-  "gopkg.in/olivere/elastic.v3",
-  "gopkg.in/olivere/elastic.v5",
-  "github.com/stretchr/*",
-  "github.com/garyburd/*",
-  "github.com/golang/*",
-  "google.golang.org/*",
-  "golang.org/x/*",
-]
-
-[[constraint]]
-  name = "github.com/ugorji/go"
-  revision = "9c7f9b7a2bc3a520f7c7b30b34b7f85f47fe27b6"
```

Comment:
can't we keep it? mostly it's to "virtually pin" the msgpack version so that it's easy to update / know what version are we running. If we keep it, we can update this value and just `dep ensure`. No?

Reply:
Sorry about that @palazzem - I wrongly assumed the dependency is a git repo. Re-added in https://github.com/DataDog/dd-trace-go/pull/162